diesel_tests/tests/expressions/mod.rs

```diff
@@ -48,7 +48,27 @@ table! {
 }
 
 #[test]
-fn test_count_max() {
+fn test_min() {
+    use self::numbers::columns::*;
+    use self::numbers::table as numbers;
+
+    let connection = connection();
+    connection
+        .execute("INSERT INTO numbers (n) VALUES (2), (1), (5)")
```

Comment:
Can we use the public API for this? `diesel::insert_into(numbers).values(&vec![n.eq(1), n.eq(2), n.eq(3)])`